File: AdvantageFramework.Core/Database/Procedures/MediaRFPHeader.cs
using System.Linq;

namespace AdvantageFramework.Core.Database.Procedures
{
    public static partial class MediaRFPHeader
    {
        #region  Constants 



        #endregion

        #region  Enum 



        #endregion

        #region  Variables 



        #endregion

        #region  Properties 



        #endregion

        #region  Methods 

        public static Database.Entities.MediaRFPHeader LoadByID(Database.DbContext DbContext, int ID)
        {
            Database.Entities.MediaRFPHeader LoadByIDRet = default;
            if ((from MediaRFPHeader in DbContext.MediaRfpHeaders.AsQueryable()
                 where MediaRFPHeader.MediaRfpHeaderId == ID
                 select MediaRFPHeader).Count() == 1)
            {
                LoadByIDRet = (from MediaRFPHeader in DbContext.MediaRfpHeaders.AsQueryable()
                               where MediaRFPHeader.MediaRfpHeaderId == ID
                               select MediaRFPHeader).Single();
            }
            else
            {
                LoadByIDRet = default;
            }

            return LoadByIDRet;
        }

        public static System.Linq.IQueryable<Entities.MediaRFPHeader> LoadByMediaBroadcastWorksheetMarketID(Database.DbContext DbContext, int MediaBroadcastWorksheetMarketID)
        {
            System.Linq.IQueryable<Entities.MediaRFPHeader> LoadByMediaBroadcastWorksheetMarketIDRet = default;
            LoadByMediaBroadcastWorksheetMarketIDRet = from MediaRFPHeader in DbContext.MediaRfpHeaders.AsQueryable()
                                                       where MediaRFPHeader.MediaBroadcastWorksheetMarketId == MediaBroadcastWorksheetMarketID
                                                       select MediaRFPHeader;
            return LoadByMediaBroadcastWorksheetMarketIDRet;
        }

        public static System.Linq.IQueryable<Entities.MediaRFPHeader> Load(Database.DbContext DbContext)
        {
            System.Linq.IQueryable<Entities.MediaRFPHeader> LoadRet = default;
            LoadRet = from MediaRFPHeader in DbContext.MediaRfpHeaders.AsQueryable()
                      select MediaRFPHeader;
            return LoadRet;
        }

        #endregion
    }
}

File: AdvantageFramework/Database/Procedures/MediaRFPHeader.vb
Namespace Database.Procedures.MediaRFPHeader

    <HideModuleName()>
    Public Module Methods

#Region " Constants "



#End Region

#Region " Enum "



#End Region

#Region " Variables "



#End Region

#Region " Properties "



#End Region

#Region " Methods "

        Public Function LoadByID(DbContext As Database.DbContext, ID As Integer) As Database.Entities.MediaRFPHeader

            If (From MediaRFPHeader In DbContext.GetQuery(Of Database.Entities.MediaRFPHeader)
                Where MediaRFPHeader.ID = ID
                Select MediaRFPHeader).Count = 1 Then

                LoadByID = (From MediaRFPHeader In DbContext.GetQuery(Of Database.Entities.MediaRFPHeader)
                            Where MediaRFPHeader.ID = ID
                            Select MediaRFPHeader).Single

            Else

                LoadByID = Nothing

            End If

        End Function
        Public Function LoadByMediaBroadcastWorksheetMarketID(DbContext As Database.DbContext, MediaBroadcastWorksheetMarketID As Integer) As System.Data.Entity.Infrastructure.DbQuery(Of Database.Entities.MediaRFPHeader)

            LoadByMediaBroadcastWorksheetMarketID = From MediaRFPHeader In DbContext.GetQuery(Of Database.Entities.MediaRFPHeader)
                                                    Where MediaRFPHeader.MediaBroadcastWorksheetMarketID = MediaBroadcastWorksheetMarketID
                                                    Select MediaRFPHeader

        End Function
        Public Function Load(DbContext As Database.DbContext) As System.Data.Entity.Infrastructure.DbQuery(Of Database.Entities.MediaRFPHeader)

            Load = From MediaRFPHeader In DbContext.GetQuery(Of Database.Entities.MediaRFPHeader)
                   Select MediaRFPHeader

        End Function

#End Region

    End Module

End Namespace
